Formula/cookiecutter.rb

```diff
@@ -14,7 +14,7 @@ class Cookiecutter < Formula
     sha256 "2add95022eec8b10c91f28bfa209ba1bd866e474a8d65350a965d081211b4584" => :el_capitan
   end
 
-  depends_on "python@2"
+  depends_on "python"
```

Comment:
Switch to Python 3, cookiecutter supports [all the way up to 3.6](https://github.com/audreyr/cookiecutter#features).